### ISSUE TITLE
How to use this repo to show Chinese characters?

### ISSUE THREAD

**Issue by zydjohnHotmail** [arklumpus/VectSharp#6]
I have the following C# code to make some text centered.
 public static void CenterTextPNG1Page(string text1, string png_file1)
 {
     Font fnt = new(new FontFamily(FontFamily.StandardFontFamilies.CourierBold), 20);
     Graphics textGraphics = new();
     double currY = fnt.FontSize * 1.4;
     if ((text1.Length >= 80) && (text1.Length <= 100))
     {
  fnt = new(new FontFamily(FontFamily.StandardFontFamilies.CourierBold), 12);
     }
     Size currentLineSize = fnt.MeasureText(text1);
     textGraphics.FillText((B20_Image_Width - currentLineSize.Width) * 0.5,
       currY, text1, fnt, Colours.Black, TextBaselines.Baseline);
     currY += fnt.FontSize * 1.4;
     Page containerPage = new(B20_Image_Width, B20_CC_Height);
     containerPage.Graphics.DrawGraphics((containerPage.Width - B20_Image_Width) * 0.5,
  (containerPage.Height - currY) * 0.5, textGraphics);
     containerPage.Background = Colours.White;
     containerPage.SaveAsPNG(png_file1, 1);
 }

For most Western languages, it works well.  But I found that I can’t use the above C# code to show Chinese characters.  The Chinese characters are shown correctly in Windows 10, and correctly in TextBox in WinForms.
But I can’t use for showing Chinese characters.
For example, if I write one simple Chinese sentence: "你好 – 世界" (Meaning: Hello - World)
string test_chinese_text = "你好 - 世界";
CenterTextPNG1Page(test_chinese_text,  @"C:\Images\Chinese.png");
Then the only result in the PNG file is the dash, which is English. 
You can see it from the picture.
![1208252_cn_cc](https://user-images.githubusercontent.com/45263518/130228710-0c0a4d37-139d-468b-bc48-826c2806d420.png)
Please advise, what I can do to show the Chinese text in the PNG file, also in the center of the the line.
Do I have to install some special font for Chinese? But I can display it correctly in Windows 10 and in TextBox of a WinForm App project.
Thanks,

**Comment by arklumpus** [comment]
Hi, first of all I must say that I have zero experience with non-Western locales, so I will not be able to help much with this.

In any case, it seems that the main problem is that the embedded fonts do not support Chinese characters; the solution is simply to use a custom font family created using a TTF font file that has Chinese characters.

For example, using a font downloaded from  [here](https://www.zcool.com.cn/special/zcoolfonts/), I obtained this:

![test](https://user-images.githubusercontent.com/32712606/130261628-8411ddf0-0104-4f56-973d-c8169ca811db.png)

To do this, you just have to replace the line where you create the font with:

```CSharp
Font fnt = new(new FontFamily("path/to/font.ttf"), 20);
```

Obviously replace `path/to/font.ttf` with the actual location of the TTF font file. An important note is that the font must be in TTF format (and not OTF), otherwise it will not work. The font I linked is a Simplified Chinese font, but I'm sure that you will be able to find a font file for Traditional Chinese if you look for it.
